fix: unable to terminate a compute session when wsproxy address is not properly (#1423)
lablup/backend.ai: changes/1423.fix.md

> Unable to terminate a compute session when wsproxy address is not properly

Reviewer: ```suggestion
Enable to terminate a compute session when wsproxy server has any connection issue.
```

Author: Thanks for reviewing! :)

> Enable to terminate a compute session when wsproxy server has any connection issue.

Reviewer 2: The sentence looks strange. "has any connection issue"??

Author: Thanks for reviewing!
I will fix it.
